lea/views/dag.py

```diff
@@ -55,9 +55,10 @@ def _to_mermaid_views(self):
             self.dependencies.keys()
         )
         schema_nodes = itertools.groupby(sorted(nodes), lambda node: node[0])
-        for schema, nodes in sorted(schema_nodes):
+        for schema, nodes in schema_nodes:
```

Comment:
I did the sorting because I wanted the output to be idempotant. If not, git tracks changes in each time. Does this really not work if you remove `sorted`?

Reply:
![image](https://github.com/carbonfact/lea/assets/36007380/e31e1d7d-d7c4-46ad-b585-d7de000b8d8e)
looks like sorting breaks the iterator.

However, the iterator should preserve the nodes order (already sorted). Or am I missing something?

Reply:
No you're right, it should be fine. Let's merge!